release-calendar/src/client/stylesheets/calendar.scss

```diff
@@ -6,6 +6,6 @@
   font-family: Arial, Helvetica Neue, Helvetica, sans-serif;
   font-size: 14px;
   margin: 0 auto;
```

Comment:
The left side of the page has more whitespace than the right side. Can that be more balanced?